### ISSUE TITLE
Function extension access to LineRunner

### ISSUE THREAD

**Issue by linux-china** [cbeust/kash-example#1]
For some function extensions, such as time, alias etc, the command will be replaced as new  command then sent to LineRunner.  Any solution to access global LineRunner from the extension?

```
class SimpleBuiltin(val lineRunner: LineRunner ) {
    @Builtin
    fun time(words: List<String>, context: IKashContext): CommandResult {
         .... measure here
        return  lineRunner.runLine(...)
    }
}
```

**Comment by cbeust** [comment]
Yes, I can add this to `IKashContext`.

**Comment by cbeust** [comment]
Fixed in Kash 1.16, there is now `IKashContext#lineRunner`.
